gui/src/components/logs/logTypes.ts

```diff
@@ -0,0 +1,12 @@
+interface Log {
+  id: string;
+  style: string;
+  time: {
+    short: string;
+    full: string;
+  };
+  name: string;
+  message: string;
+}
+
+export type Logs = Log[];
```

Comment:
Generally, type aliases that are just `T[]` are not all that useful and imply that there is more to the type.

Reply:
Fixed, thanks :)